src/jit/lclvars.cpp

```diff
@@ -3444,6 +3445,17 @@ var_types LclVarDsc::lvaArgType()
                 break;
         }
     }
+#else  // FEATURE_UNIX_AMD64_STRUCT_PASSING
+    if (type == TYP_STRUCT)
+    {
+        NYI("lvaArgType");
+    }
+#endif // !FEATURE_UNIX_AMD64_STRUCT_PASSING
+#elif defined(_TARGET_ARM64_)
```

Comment:
Two nits:
1. It is generally preferable when an `#if` has an `#else` to put the "positive" (non-negated) condition first, to reduce potential confusion resulting from reasoning about double negatives (and potentially missing the "n" in "ifndef")
2. Coding convention states the comment on a `#else` should indicate the condition of the code that follows the `#else`, whereas the comment on the `#endif` indicates the condition of the code that precedes the `#endif`. Thus:
```
#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
...
#else  // !FEATURE_UNIX_AMD64_STRUCT_PASSING
...
#endif // !FEATURE_UNIX_AMD64_STRUCT_PASSING
```